docs/source/conf.py

```diff
@@ -404,6 +404,7 @@
     # torch.cuda.memory
     "caching_allocator_alloc",
     "caching_allocator_delete",
+    "caching_allocator_enable",
```

Comment:
Adding new entries to this file is NOT ok.
If you have to add this, it means that the API is not properly documented on the website and your change to the .rst file is not doing what you expect!

Given your change below, you most likely just need to remove this line!

Reply:
Removed that line.  Waiting for new CI signal for docs.